### DISCUSSION TITLE
186B0D65-7BC3-BBCD-BB72-58C07BE4C292

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#64] [General]
# 186B0D65-7BC3-BBCD-BB72-58C07BE4C292

This addon shows moments from the science fiction-romance movie Passengers, a by Morten Tyldum and Jon Spaihts, released in 2016. In the movie, the interstellar spaceship Avalon, with 268 crew and 5,000 passengers, is destined for the planet Homestead II, on a journey of 120 years. Crew and passengers are in suspended animation (hibernation).
In this addon you can see the moment that Avalon is leaving the Earth, the moment that it uses impulse with the star Arcturus and the arrival on the planet Homestead II. You can also explore the planetary system around the Homestead star.

https://celestia.mobi/addon?item=186B0D65-7BC3-BBCD-BB72-58C07BE4C292&lang=zh_CN

**Comment by 13940951572** [comment]
这个飞船怎么启动！
